### ISSUE TITLE
使用train_grpo_ray_hybrid_engine.sh的时候日志没有vllm的生成cuda graph正常吗

### ISSUE THREAD

**Issue by heningsu** [OpenRLHF/OpenRLHF#1008]
因推理速度太慢，检查了很多其他原因之后怀疑没正确使用vllm，附上我的脚本set -x

ray job submit --address="http://octave:8265" \
   --runtime-env-json='{"working_dir": "/home/suliuhui/OpenRLHF"}' \
   -- python3 -m openrlhf.cli.train_grpo_ray \
   --ref_num_nodes 1 \
   --ref_num_gpus_per_node 2 \
   --actor_num_nodes 1 \
   --actor_num_gpus_per_node 2 \
   --vllm_num_engines 2 \
   --vllm_tensor_parallel_size 1 \
   --vllm_gpu_memory_utilization 0.7 \
   --init_kl_coef 1e-3 \
   --gamma 1.0 \
   --use_kl_loss \
   --kl_estimator k3 \
   --advantage_estimator group_norm \
   --pretrain /home/suliuhui/Qwen2.5-3B-Instruct \
   --remote_rm_url  examples/scripts/reward_func.py \
   --save_path /home/suliuhui/OpenRLHF/checkpoint  \
   --ckpt_path /home/suliuhui/OpenRLHF/checkpoint \
   --save_hf_ckpt \
   --micro_train_batch_size 4 \
   --train_batch_size 32 \
   --micro_rollout_batch_size 2 \
   --rollout_batch_size 8 \
   --n_samples_per_prompt 4 \
   --max_epochs 1 \
   --prompt_max_len 1024 \
   --max_samples 100000 \
   --generate_max_len 1024 \
   --zero_stage 3 \
   --bf16 \
   --actor_learning_rate 5e-6 \
   --prompt_data /home/suliuhui/OpenRLHF/medical.jsonl \
   --input_key prompt \
   --label_key answer \
   --normalize_reward \
   --gradient_checkpointing \
   --packing_samples \
   --vllm_sync_backend nccl \
   --enforce_eager \
   --save_steps 3 \
   --use_tensorboard /home/suliuhui/OpenRLHF/tensor-grpo-1 \
   --save_hf_ckpt \
   --vllm_enable_sleep \
   --use_wandb {} \
   --deepspeed_enable_sleep

**Comment by hijkzzz** [comment]
去掉 --enforce_eager
